lib/OpenQA/WebAPI/Command/gru/list.pm

```diff
@@ -0,0 +1,94 @@
+
+    script/openqa gru list
+    script/openqa gru list 10023
+    script/openqa gru list -w
+    script/openqa gru list -w 23
+    script/openqa gru list -s
+    script/openqa gru list -f 10023
+    script/openqa gru list -q important -t foo -t bar -S inactive
+    script/openqa gru list -e foo -a '[23, "bar"]'
+    script/openqa gru list -e foo -P 10023 -P 10024 -p 5 -q important
+    script/openqa gru list -R -d 10 10023
+    script/openqa gru list --remove 10023
+    script/openqa gru list -L
+    script/openqa gru list -L some_lock some_other_lock
+    script/openqa gru list -b jobs -a '[12]'
+    script/openqa gru list -b jobs -a '[12]' 23 24 25
+
+  Options:
+    -A, --attempts <number>     Number of times performing this new job will be
+                                attempted, defaults to 1
+    -a, --args <JSON array>     Arguments for new job or worker remote control
+                                command in JSON format
+    -b, --broadcast <command>   Broadcast remote control command to one or more
+                                workers
+    -d, --delay <seconds>       Delay new job for this many seconds
+    -e, --enqueue <task>        New job to be enqueued
+    -f, --foreground            Retry job in "minion_foreground" queue and
+                                perform it right away in the foreground (very
+                                useful for debugging)
+    -H, --history               Show queue history
+    -h, --help                  Show this summary of available options
+        --home <path>           Path to home directory of your application,
+                                defaults to the value of MOJO_HOME or
+                                auto-detection
+    -L, --locks                 List active named locks
+    -l, --limit <number>        Number of jobs/workers to show when listing
+                                them, defaults to 100
+    -m, --mode <name>           Operating mode for your application, defaults to
+                                the value of MOJO_MODE/PLACK_ENV or
+                                "development"
+    -o, --offset <number>       Number of jobs/workers to skip when listing
+                                them, defaults to 0
+    -P, --parent <id>           One or more jobs the new job depends on
+    -p, --priority <number>     Priority of new job, defaults to 0
+    -q, --queue <name>          Queue to put new job in, defaults to "default",
+                                or list only jobs in these queues
+    -R, --retry                 Retry job
+        --remove                Remove job
+    -S, --state <name>          List only jobs in these states
+    -s, --stats                 Show queue statistics
+    -t, --task <name>           List only jobs for these tasks
+    -U, --unlock <name>         Release named lock
+    -w, --workers               List workers instead of jobs, or show
+                                information for a specific worker
+
```

Comment:
Looks like you've just copied the lines above from somewhere else. Wouldn't it be sufficient just to put a link to that place so we don't have to care to keep it in sync?

Reply:
This is not just documentation, the options get shown when you do `script/openqa gru list -h`.

Reply:
If that can technically not easily be solved without so much redundancy it is ok.